ui/eslint.config.js

```diff
@@ -8,12 +8,7 @@ const config = defineConfig([
   ...createConfig(opts),
   {
     rules: {
-      'jsx-a11y/anchor-is-valid': 'off',
```

Comment:
Forbids using `<a>` for interactive elements that are not actually links and should be `<button>`s instead. Not trivial to do from a styling point of view, and since the gain is minimal (accessibility), it's okay to not fix existing violations right away. Of course, if you refactor a component with such a violation, do take the time to fix it. Same applies to all the rules below.